packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.component.js

```diff
@@ -46,8 +46,9 @@ function ListToVirtualizedList(props) {
 		>
 			{props.columns.map((column, index) => {
 				const cProps = {
-					label: column.label,
+					label: column.hideHeader ? '' : column.label,
```

Comment:
Instead of setting a blank label, you should defined a specific `headerRenderer` and return something like `<span className="sr-only">{column.label}</span>`. This is far better for accessibility.